packages/fast-jss-manager-react/README.md

```diff
@@ -1,2 +1,67 @@
+# React JSS Manager
+`fast-react-jss-manager` is a React higher order component (HOC) library for managing component JSS stylesheets. It facilitates using JSS with your react components by controlling when to add, update, and remove a JSS stylesheets. It also allows top-down variable injection into JSS stylesheet functions using the exported `DesignSystemProvider`.
 
-A React higher order component (HOC) library for managing component JSS stylesheets.
+
+## Usage
+### Create a react component
+`manageJss` is a HOC that passes as props the generated classes from a given stylesheet. The generated classes are made available on the `managedClasses` prop of the rendered component.
+```jsx
+// button.js
+import * as React from "react";
+
+class Button extends React.Component {
+    render(): {
```

Comment:
you have a colon, do you want to say this is tsx and return JSX.Element?

Reply:
resolved